veracruz-server/src/cli.rs

```diff
@@ -20,12 +20,70 @@ use veracruz_utils::policy::{
 };
 
 
+/// A bit of extra parsing to allow omitting addr/port
+fn parse_bind_addr(s: &str) -> String {
```

Comment:
Is this function repeated?  Maybe move it out into e.g. `veracruz_utils` or similar.